user-service/src/main/java/com/movie/recommender/user/model/dto/AddFavoriteMoviesDTO.java

```diff
@@ -0,0 +1,14 @@
+package com.movie.recommender.user.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class AddFavoriteMoviesDTO {
```

Comment:
rename to a more neutral name like `FavoriteMoviesDTO`, as it is used for both adding and removing favorite movies